bundles/com.espressif.idf.terminal.connector.serial/src/com/espressif/idf/terminal/connector/serial/launcher/CoreDumpPostmortemDebuggerLauncher.java

```diff
@@ -0,0 +1,272 @@
+		final String booleanTrue = "true"; //$NON-NLS-1$
+
+		DocumentBuilder builder = DocumentBuilderFactory.newInstance().newDocumentBuilder();
+		Document dom = builder.newDocument();
+
+		Element root = dom.createElement("launchConfiguration"); //$NON-NLS-1$
+		dom.appendChild(root);
+		Attr attr = dom.createAttribute("type"); //$NON-NLS-1$
+		attr.setValue("org.eclipse.cdt.launch.postmortemLaunchType"); //$NON-NLS-1$
+		root.setAttributeNode(attr);
+
+		createElement(dom, root, booleanAttribute, "org.eclipse.cdt.dsf.gdb.AUTO_SOLIB", booleanTrue); //$NON-NLS-1$
+		createElement(dom, root, listAttribute, "org.eclipse.cdt.dsf.gdb.AUTO_SOLIB_LIST", null); //$NON-NLS-1$
+		createElement(dom, root, stringAttribute, "org.eclipse.cdt.dsf.gdb.DEBUG_NAME", //$NON-NLS-1$
+				IDFUtil.getXtensaToolchainExecutablePath(project));
+		createElement(dom, root, stringAttribute, "org.eclipse.cdt.dsf.gdb.GDB_INIT", ""); //$NON-NLS-1$ //$NON-NLS-2$
+		createElement(dom, root, stringAttribute, "org.eclipse.cdt.dsf.gdb.POST_MORTEM_TYPE", "CORE_FILE"); //$NON-NLS-1$//$NON-NLS-2$
+		createElement(dom, root, listAttribute, "org.eclipse.cdt.dsf.gdb.SOLIB_PATH", null); //$NON-NLS-1$
+		createElement(dom, root, intAttribute, "org.eclipse.cdt.launch.ATTR_BUILD_BEFORE_LAUNCH_ATTR", //$NON-NLS-1$
+				String.valueOf(100));
+
+		createElement(dom, root, stringAttribute, "org.eclipse.cdt.launch.COREFILE_PATH", //$NON-NLS-1$
+				project.getFile(GENERATED_CORE_ELF_NAME).getRawLocation().toOSString());
+
+		createElement(dom, root, stringAttribute, "org.eclipse.cdt.launch.DEBUGGER_ID", "gdb"); //$NON-NLS-1$ //$NON-NLS-2$
+		createElement(dom, root, stringAttribute, "org.eclipse.cdt.launch.DEBUGGER_START_MODE", "core"); //$NON-NLS-1$ //$NON-NLS-2$
+		createElement(dom, root, stringAttribute, "org.eclipse.cdt.launch.PROGRAM_NAME", elfFilePath); //$NON-NLS-1$
+		createElement(dom, root, stringAttribute, "org.eclipse.cdt.launch.PROJECT_ATTR", project.getName()); //$NON-NLS-1$
+		createElement(dom, root, booleanAttribute, "org.eclipse.cdt.launch.PROJECT_BUILD_CONFIG_AUTO_ATTR", //$NON-NLS-1$
+				booleanFalse);
+		createElement(dom, root, stringAttribute, "org.eclipse.cdt.launch.PROJECT_BUILD_CONFIG_ID_ATTR", //$NON-NLS-1$
+				"org.eclipse.cdt.core.default.config.2083943554"); //$NON-NLS-1$
+
+		Element listMappResPath = dom.createElement(listAttribute);
+		listMappResPath.setAttribute(key, "org.eclipse.debug.core.MAPPED_RESOURCE_PATHS"); //$NON-NLS-1$
+		Element subListEntryResPath = dom.createElement(listEntry);
+		subListEntryResPath.setAttribute(value, "/".concat(project.getName())); //$NON-NLS-1$
+		listMappResPath.appendChild(subListEntryResPath);
+		root.appendChild(listMappResPath);
+
+		Element listMappResType = dom.createElement(listAttribute);
+		listMappResType.setAttribute(key, "org.eclipse.debug.core.MAPPED_RESOURCE_TYPES"); //$NON-NLS-1$
+		Element subListEntryResType = dom.createElement(listEntry);
+		subListEntryResType.setAttribute(value, "4"); //$NON-NLS-1$
```

Comment:
To improve performance and reduce the code maintenance could we bundle this file with our plugins and copy this launcher file to the project and update the required attributes(.elf and project name)